static/js/app.ts

```diff
@@ -39,6 +39,7 @@ let askPromptOpen = false;
 let theAdventures: Record<string, Adventure> = {};
 let theLevel: number = 0;
 let theLanguage: string = '';
+let theKeywordLanguage: string = 'en';
```

Comment:
Why do we initialise language to '' but theKeywordLanguage to 'en'? Seems a bit weird, should we explain this?